mysql-test/columnstore/bugfixes/mcol-5476.result

```diff
@@ -0,0 +1,71 @@
+DROP DATABASE IF EXISTS `mcol-5476`;
+CREATE DATABASE `mcol-5476`;
+USE `mcol-5476`;
+create table t1 (a int, b int) engine=columnstore;
+insert into t1 values (1, 1), (2, 1), (3, 1), (4, 2), (5, 2);
+select sum(a), abs(b), abs(b) from t1 group by abs(b), abs(b) order by abs(b);
+sum(a)	abs(b)	abs(b)
+6	1	1
+9	2	2
+select sum(a), abs(b), abs(b) from t1 group by abs(b) order by abs(b);
+sum(a)	abs(b)	abs(b)
+6	1	1
+9	2	2
+select sum(distinct a), abs(b), abs(b) from t1 group by abs(b), abs(b) order by abs(b);
```

Comment:
You should use #sorted_result MTR interpreter clause. This way MTR sorts the given and the expected outputs on its own before comparison. You can drop ORDER BY using #sorted_result.

Reply:
This is valid for all similar cases in this test.